src/d7/gitignore

```diff
@@ -15,3 +15,8 @@ secret.settings.php
 local.settings.php
 
 .aquifer/node_modules
+aquifer.local.json
+
+.DS_Store
+Thumbs.db
+*.sql
```

Comment:
reference databases... but really... this also be a project by project basis. There may be a perfectly valid use-case for having sql files in the repository.